mmv1/templates/terraform/iam/example_config_body/cloudfunctions2_function.tf.erb

```diff
@@ -1,2 +0,0 @@
-  cloud_function = google_cloudfunctions2_function.%{primary_resource_id}.name
```

Comment:
not sure what was the motivation for this but it works fine without it

Reply:
Agreed.